Sources/SpotHeroAPI/Search/Transient/Requests/SearchGetTransientFacilitiesRequest.swift

```diff
@@ -69,6 +71,12 @@ public extension SearchGetTransientFacilitiesRequest {
         /// result distances are calculated from the required `latitude` and `longitude` parameters. Origin longitude must be in [-180, 180].
         private let originLongitude: Double?
         
+        /// This represents the work address latitude associated with the user’s commuter benefits card. Latitude must be in [-90, 90].
```

Comment:
I feel like `This represents` could be removed from these. (Otherwise, I'd feel like we should say `This represents ...` on all properties to be consistent.

```suggestion
        /// The work address latitude associated with the user’s commuter benefits card. Latitude must be in [-90, 90].
```

Reply:
Agreed, it's taken from the BE docs so I'll propose the change back to them as well. 👍🏻